### PR TITLE
Dw 5339 migrate telephone claims

### PR DESCRIPTION
**Pull request by phanindrawork**

### CODE REVIEW

File: ci/aws-analytical-env/shared/meta.yml [hunk 1]
```diff
@@ -349,3 +350,184 @@ meta:
...
+    terraform-bootstrap-crown:
+      task: terraform-bootstrap-crown
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli.repository))
+            version: ((docker-awscli.version))
+            tag: ((docker-awscli.version))
```
Comment (MattToczek) [inline ci/aws-analytical-env/shared/meta.yml:363]: should these not be `docker.docker_awscli_repository`, `docker.docker_awscli_version`?

File: ci/aws-analytical-env/shared/meta.yml [hunk 2]
```diff
@@ -349,3 +350,184 @@ meta:
...
+    create-aws-profiles:
+      task: create-aws-profiles
+      attempts: 15
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli-ubuntu.repository))
+            version: ((docker-awscli-ubuntu.version))
```
Comment (MattToczek) [inline ci/aws-analytical-env/shared/meta.yml:392]: same as above

File: ci/aws-analytical-env/shared/meta.yml [hunk 3]
```diff
@@ -349,3 +350,184 @@ meta:
...
+    terraform-output-adg:
+      task: terraform-output-adg
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((terraform.repository))
+            version: ((terraform12.version))
+            tag: ((terraform12.version))
```
Comment (MattToczek) [inline ci/aws-analytical-env/shared/meta.yml:459]: Should this be `docker.terraform_repository` and `docker.terraform_12_version`?

File: ci/aws-analytical-env/shared/meta.yml (hunk 1 above)
Comment (phanindrawork) [reply ci/aws-analytical-env/shared/meta.yml:363]: Previously was using `repository: dwpdigital/jinja-yaml-aws` but this failed during run. Had a chat with @dan-hill2802  he suggested to use `docker-awscli.repository`

**Review by MattToczek — APPROVED** [review]

**Pull request merged**